### DISCUSSION TITLE
Can't activate line numbers

### DISCUSSION THREAD

**Discussion by Yann-C-INN** [dbeaver/dbeaver#20136] [Q&A] [answered]
### Description

Hey guys. I would think this option should be on as default, but anyway,
I tried to activate line numbers but it doesnt seem to work
![image](https://github.com/dbeaver/dbeaver/assets/130037694/4fae6df1-8b73-4d12-8a73-b80913cec26d)
I clicked to show line numbers as global options as well as my current connections, and restareted dbeaver after, but it still doesn't show line numbers (and then when i have an error in line 125, go figure).

Would appreciate your help :)

### DBeaver Version

23.0.1

### Operating System

Windows 11 

### Database and driver

Azure

### Steps to reproduce

Alt + Enter
Editors
Data editor
Appearance
Plain text
2nd line checkbox
Restart
still doesn't work

**Comment by E1izabeth** [comment, marked as answer]
I suppose you are looking for Show line number setting for SQL editor, not for data editor. In this case you need to turn of the corresponding setting in Preferences->Editors->Text Editors.
![image](https://github.com/dbeaver/dbeaver/assets/28875055/0412201f-fe72-4103-a991-0e366a057f0a)

Please, let me know if it helps.

**Comment by Yann-C-INN** [comment]
> I suppose you are looking for Show line number setting for SQL editor, not for data editor. In this case you need to turn of the corresponding setting in Preferences->Editors->Text Editors. ![image](https://user-images.githubusercontent.com/28875055/241986952-0412201f-fe72-4103-a991-0e366a057f0a.png)
> 
> Please, let me know if it helps.

Thank yoi Elizabeth. I do not have that option.
I only have SQL editor and Data editor. 

OK. big UI/UX issue guys :) There is both Properties AND preferences. I was at the properties window. I only then used the search option and discovered there is actually a preference one.

Thanks for helping, and I am a bit a shamed i didnt find it myself (though seriously, fix it. why two of them?)

**Comment by E1izabeth** [comment]
Probably, you are looking for that in connection setting. Could you try opening Global preferences?
![image](https://github.com/dbeaver/dbeaver/assets/28875055/4ffb142a-63c8-4c94-ad2d-58f08d07d01a)


or editor preferences
![image](https://github.com/dbeaver/dbeaver/assets/28875055/77580051-d091-4afc-a248-3e75e7017ddf)

**Comment by Yann-C-INN** [comment]
> 

Sorry, I edited my response. you indeed helped me (and i indeed feel a bit stupid for not finding it myself).
THANKS!
